src/main/java/core/basesyntax/Obtainable.java

```diff
@@ -0,0 +1,3 @@
+package core.basesyntax;
+
+public interface Obtainable { double obtain(); }
```

Comment:
Obtainable is too vague of a name, go with AreaCalculator, interfaces don't all need to be adjectives